tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Conventions/UnifiedPipelineConvention.cs

```diff
@@ -19,7 +19,7 @@ protected override string GetDefinitionName(SdkComponent component)
             return component.Variant == null ? $"{Context.Prefix} - {component.Name}" : $"{Context.Prefix} - {component.Name} - {component.Variant}";
         }
 
-        public override string SearchPattern => "ci*.yml";
+        public override string SearchPattern => "ci.yml";
```

Comment:
I added the variant detection directly in the scanning tool because this "*" was actually picking up other files when I enabled it for tests (i.e. tests-install-azurite.yml) and so I realized this may be a little looser then we want. I made the pattern always require "<name>.<variant>.yml" or "<name>.yml" now.